Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:trusty
```

Comment:
There's a official clickhouse image on dockerhub - `yandex/clickhouse-server`, and configs could be passed using volumes.